### DISCUSSION TITLE
Next to Visibility, add Editor Visibility

### DISCUSSION THREAD

**Discussion by philippedcote** [godotengine/godot-proposals#7598] [GUI]
While working on complex UI projects, the designer shouldn't have to worry about breaking the project by setting the visibility of objects to different settings than what the programmer needs it to be.

For these cases, we should add an Editor Visibility next to the Visibility icon to be able to display an object without fearing of breaking the project.

Here is a draft below. Unsure about the icon but you get the idea. Or maybe the Check and Cross should replace the current "Visible" parameter, and keep the Eye icon for Editor Visibility.

![image](https://github.com/godotengine/godot-proposals/assets/890553/dd136b32-1d1b-4785-a9d1-0fadcb3ee6d7)

As an example, Cinema 4D manage visibility like this, which very, very convenient:
![image](https://github.com/godotengine/godot-proposals/assets/890553/676981fc-876a-4ab4-a899-822948bc5ef1)

**Comment by Calinou** [comment]
- Related to https://github.com/godotengine/godot-proposals/issues/3433.
